### PR TITLE
APP Datetime Format

### PR DESCRIPTION
**Pull request by damonkelley**
`last_verified` and `priority_change_date` need t be in the ISO 8601 format.

### CODE REVIEW

**Comment by damonkelley** [comment]
ping @ldko @somexpert

File: coda/coda_validate/views.py [hunk 1]
```diff
@@ -1,30 +1,21 @@
...
-    # http://pypi.python.org/pypi/simplejson/2.0.9
-    # simplejson 1.7.3 is available for python 2.3+
-    # http://pypi.python.org/pypi/simplejson/1.7.3
-    import simplejson as json
-
-from django.shortcuts import render_to_response
-from django.template import RequestContext
-from django.core.paginator import Paginator
+
+import datetime
```
Comment (somexpert) [inline coda/coda_validate/views.py]: Think this import belongs in the first group.

Comment (damonkelley) [inline coda/coda_validate/views.py]: yeah, i didn't spend any time scrutinizing the output. 

>Some of the codalib imports are split across two groups. I'm thinking rope did this, right?

Not sure what you mean here though.

Comment (somexpert) [inline coda/coda_validate/views.py]: Ah, realized I was reading it wrong. codalib imports are fine.

**Comment by somexpert** [comment]
:thumbsup:

**Comment by ldko** [comment]
:+1:

**Pull request merged**